tests/CommaSeparatedTypeParserTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Test\Common\Foundation\RequestSpec;
+
+use MPScholten\RequestParser\CommaSeparatedTypeParser;
+
+class CommaSeparatedTypeParserTest extends \PHPUnit_Framework_TestCase
+{
+    private function createExceptionFactory()
+    {
+        return function() {
+            throw new NotFoundException();
+        };
+    }
+
+    public function testCsvWithInt()
```

Comment:
Good job with the tests, [100% code coverage](https://68-61488582-gh.circle-artifacts.com/0//tmp/circle-artifacts.QqzQ7gl/CommaSeparatedTypeParser.php.html) 👍